pom.xml

```diff
@@ -554,6 +554,7 @@
       <groupId>com.google.inject</groupId>
       <artifactId>guice</artifactId>
       <version>4.2.3</version>
+      <classifier>no_aop</classifier>
```

Comment:
See https://github.com/google/guice/issues/1133#issuecomment-704281989 for why this is necessary